[ao][sparsity] README for base data scheduler class

macandro96 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #82131
* #82130
* #82129

The readme file contains an overview of the base data scheduler.
Consists of code snippets and instructions on how to create your own custom
data scheduler and how to use during training a model.

Test Plan: None

Commit:
[ao][sparsity] README for base data scheduler class

The readme file contains an overview of the base data scheduler.
Consists of code snippets and instructions on how to create your own custom
data scheduler and how to use during training a model.

Test Plan: None

[ghstack-poisoned]

Commit:
Update on "[ao][sparsity] README for base data scheduler class"


The readme file contains an overview of the base data scheduler.
Consists of code snippets and instructions on how to create your own custom
data scheduler and how to use during training a model.

Test Plan: None

[ghstack-poisoned]

z-a-f reviewed (approved):
LGTM, but add spaces around the code blocks

Commit:
Update on "[ao][sparsity] README for base data scheduler class"


The readme file contains an overview of the base data scheduler.
Consists of code snippets and instructions on how to create your own custom
data scheduler and how to use during training a model.

Test Plan: None

[ghstack-poisoned]

Commit:
Update on "[ao][sparsity] README for base data scheduler class"


The readme file contains an overview of the base data scheduler.
Consists of code snippets and instructions on how to create your own custom
data scheduler and how to use during training a model.

Test Plan: None

[ghstack-poisoned]

macandro96 commented:
@pytorchbot merge -g
